components/gitpod-db/src/user-to-team-migration-service.ts

```diff
@@ -144,7 +144,6 @@ export class UserToTeamMigrationService {
     }
 
     async needsMigration(user: User): Promise<boolean> {
-        const teams = await this.teamDB.findTeamsByUser(user.id);
-        return teams.length === 0 || !user.additionalData?.isMigratedToTeamOnlyAttribution;
+        return !user.additionalData?.isMigratedToTeamOnlyAttribution;
```

Comment:
:+1:

Reply:
FWIW: This case can happen in situations where a user deletes their last org or a user is kicked out of their last org.
In those cases we need to require creating an org (or use an invite link to join one) so I thought we could use the "user has no orgs" condition in general to show the first step of the "dedicated onboarding flow".

Reply:
> I thought we could use the "user has no orgs" condition in general to show the first step of the "dedicated onboarding flow"

Why not show the `/org/new` view? :thinking:

Reply:
We can of course. I just wished we wouldn't have two different ways and UIs to create an org, but I might have expressed that preference already 😇